ee/ucp/kubernetes/storage/use-vsphere-volumes.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Configuring vSphere Volumes for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters using vSphere Volumes.
+keywords: Universal Control Plane, UCP, Docker Enterprise, Kubernetes, storage, volume
+---
+
+The [vSphere Storage for Kubernetes driver](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html) enables customers to address persistent storage requirements for Kubernetes pods  in vSphere environments. The driver allows you to create a persistent  volume on a Virtual Machine File System (VMFS), and use it to manage persistent storage requirements independent of pod and VM lifecycle.
+
+> Note
+>
+> Of the three main storage backends offered by vSphere on Kubernetes (VMFS, vSAN, and NFS), we support VMFS.
+
+You can use vSphere Cloud Provider to manage storage with Kubernetes in UCP 3.1 and later. This includes support for :
+
+* Volumes
+* Persistent volumes
+* Storage classes and provisioning volumes
+
+
+## Prerequisites
+* Ensure that vsphere.conf is populated per the [vSphere Cloud Provider Configuration Deployment Guide](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html#create-the-vsphere-cloud-config-file-vsphereconf).
```

Comment:
format vsphere.conf
suggest replacing `per the` with `as per the` or 'according to the'